word_guess.rb

```diff
@@ -0,0 +1,127 @@
+require "colorize"
+require "rainbow"
+require "artii"
+
+class WordGuess
+  attr_reader :word, :wrong_letters, :dashes
+
+  def initialize
+    @word = get_word
+    @dashes = dashes_array
+    @wrong_letters = [] #empty array to hold used letters
+  end
+
+  def get_word #using this method to use in initialize to call array
+    list_of_words = ["development", "digestion", "window", "yesterday", "separate", "disease", "carriage", "punishment", "transport", "tomorrow", "instrument", "competition", "representative",
+      "achievement", "stomach", "observation", "tendency", "building", "business", "attention", "attraction", "behavior", "questionnaire", "susceptible", "lollipop", "elephant", "hotdog", "awesome",
+      "kilometer", "treasure", "dominoes", "electricity", "lightsaber", "battery", "circumference", "eloquent", "impeccable", "meticulous", "predilection", "superfluous", "vociferous", "zenith"].shuffle #shuffle the array so it's different each time
```

Comment:
I see the `.shuffle` call at the end here, but it would be easier to read if you added this method call on a separate line below